ensysmod/api/endpoints/authentication.py

```diff
@@ -1,58 +1,36 @@
-from datetime import timedelta
-
 from fastapi import APIRouter, Depends, HTTPException, status
 from fastapi.security import OAuth2PasswordRequestForm
 from sqlalchemy.orm import Session
 
-from ensysmod import schemas, crud, model
+from ensysmod import crud
 from ensysmod.api import deps
-from ensysmod.core import security, settings
+from ensysmod.model import User
+from ensysmod.schemas import Token, UserCreate, UserSchema
 
 router = APIRouter()
 
 
-@router.post("/login", response_model=schemas.Token)
-def login(
-        db: Session = Depends(deps.get_db),
-        form_data: OAuth2PasswordRequestForm = Depends()
-) -> schemas.Token:
+@router.post("/login", response_model=Token)
+def login(db: Session = Depends(deps.get_db), form_data: OAuth2PasswordRequestForm = Depends()):
```

Comment:
why did we remove the return types here?

Reply:
Simply for uniformity. I removed all return types and specified the response_model instead. Most of the endpoint functions return a model instead of a schema, but it will always be converted to a schema by FastAPI. Here it just so happens that the function already returns a schema.

Reply:
Thats fine